google/cloud/bigtable/client.py

```diff
@@ -542,32 +581,41 @@ async def row_exists(
         """
         raise NotImplementedError
 
-    async def sample_keys(
+    async def sample_row_keys(
         self,
         *,
-        operation_timeout: int | float | None = 60,
-        per_sample_timeout: int | float | None = 10,
-        per_request_timeout: int | float | None = None,
+        operation_timeout: int | float | None = None,
```

Comment:
Why no attempt timeout?

Reply:
go/cbt-client-debugging-steps#client-retry-settings needs to be updated :P